node/config.ts

```diff
@@ -123,8 +124,12 @@ const logConfigError = (func: EdgeFunction, exitCode: number, stderr: string, lo
       break
 
     case ConfigExitCode.InvalidDefaultExport:
-      throw new Error(
-        `Default export in '${func.path}' must be a function. More on the Edge Functions API at https://ntl.fyi/edge-api.`,
+      throw wrapBundleError(
+        new Error(
+          `Default export in '${func.path}' must be a function. More on the Edge Functions API at https://ntl.fyi/edge-api.`,
+        ),
+        // empty string because it's not associated with a bundling format
+        { format: '' },
```

Comment:
Rather than specifying this, could we make the options parameter in `wrapBundleError` optional?

I would go as far as making the options parameter in the `BundleError` constructor itself optional, and I'd call it directly from here:

```ts
throw new BundleError(`Default export in '${func.path}' must be a function. More on the Edge Functions API at https://ntl.fyi/edge-api.`)
```

I think the purpose of `wrapBundleError` is to wrap something that may or may not be an error. In this case we know for a fact that we're dealing with an error because we're the ones creating it.

Reply:
done